mnefun/_mnefun.py

```diff
@@ -213,7 +233,9 @@ def __init__(self, tmin=None, tmax=None, t_adjust=0, bmin=-0.2, bmax=0.0,
                  cov_method='empirical', ssp_eog_reject=None,
                  ssp_ecg_reject=None, baseline='individual',
                  reject_tmin=None, reject_tmax=None,
-                 lp_trans=0.5, hp_trans=0.5, movecomp='inter'):
+                 lp_trans=0.5, hp_trans=0.5, movecomp='inter',
+                 sss_type='maxfilter', int_order=3, ext_order=6, st_duration=10.0,
+                 st_correlation=0.98):
```

Comment:
I don't think these defaults don't match what we've been using for the MaxFiltered version of the data, can you check the run_sss.sh script? It's probably worth noting in the docs above that some of these params are only used with the Python version of the algorithm.